examples/eks-complete/outputs.tf

```diff
@@ -0,0 +1,21 @@
+# RDS
+output "rds_cluster_master_password" {
```

Comment:
This output should be marked as `sensitive = true`. There should be no reason to not use it in this module.